public-docs/recommended-tools.md

```diff
@@ -0,0 +1,81 @@
+---
+id: recommended-tools
+title: Recommended Tools
+---
+
+If you are interested in being able to quickly debug and develop Reaction, consider installing these useful tools to help you.
+
+## General
+
+- You should use [nvm](https://github.com/creationix/nvm) to install and manage NodeJS.
```

Comment:
Good to know. I didn't realize nvm was still the recommended way to install node.